engine-tests/src/tests/erc20_connector.rs

```diff
@@ -109,15 +109,16 @@ impl test_utils::AuroraRunner {
         self.make_call("submit", origin, rlp::encode(&input).to_vec())
     }
 
-    pub fn deploy_erc20_token(&mut self, nep141: &String) -> RawAddress {
+    pub fn deploy_erc20_token(&mut self, nep141: &String) -> Address {
         let result = self.make_call("deploy_erc20_token", origin(), nep141.try_to_vec().unwrap());
 
         result.check_ok();
 
-        Vec::<u8>::try_from_slice(result.value().as_slice())
+        let raw_address: [u8; 20] = Vec::<u8>::try_from_slice(result.value().as_slice())
             .unwrap()
             .try_into()
-            .unwrap()
+            .unwrap();
+        Address::from_slice(&raw_address)
```

Comment:
Could have a convenience function for taking `[u8; 20]` instead of converting to slice.

```rust
impl Address {
    pub fn from_array(array: [u8; 20]) -> Self { Self(H160(array)) }
}
```

Reply:
Why not:
```
    pub fn from_array(array: &[u8; 20]) -> Self {
        Self(H160(*array))
    }
```

Reply:
`[u8; 20]` implements `Copy` so passing by reference doesn't really save anything.